Projekt zalecenia Raportowanie stopnia zgodności rozwiązania (#299)
Siec-Dostepnosci-Cyfrowej/sdc: documentation/docs/zaopatrzenie/raportowanie-stopnia-zgodnosci-rozwiazania/raportowanie-stopnia-zgodnosci-rozwiazania.md

---

## 2. Zalecenie

Organizacja wymaga od wykonawców przekazania raportu stopnia zgodności rozwiązania z wymaganiami dostępności cyfrowej.

Raport przedstawia właściwości dostępności rozwiązania, opisuje sposób realizacji mających zastosowanie wymagań dostępności, określa stopień ich spełnienia oraz wskazuje materiały dowodowe stanowiące podstawę przedstawionych ocen.

---

## 3. Rekomendacje

### 3.1. Określanie obowiązku przekazania raportu

Organizacja określa w dokumentach zamówienia obowiązek przekazania raportu stopnia zgodności jako jednego z rezultatów realizacji zamówienia, jeżeli przedmiot zamówienia obejmuje rozwiązanie cyfrowe podlegające wymaganiom dostępności.

### 3.2. Zakres raportu

Raport obejmuje wszystkie mające zastosowanie wymagania dostępności objęte zakresem zamówienia.

Dla każdego wymagania raport zawiera:

- ocenę stopnia spełnienia wymagania;
- opis sposobu realizacji wymagania, właściwości rozwiązania mających znaczenie dla jego dostępności oraz znanych ograniczeń;

> - wskazanie materiałów dowodowych stanowiących podstawę oceny.

Reviewer: Warto rozważyć podanie przykładów materiałów dowodowych, które mogą stanowić podstawę oceny, np. wyników testów manualnych, testów z wykorzystaniem technologii asystujących, raportu z audytu czy dokumentacji technicznej. Ułatwiłoby to wykonawcy przygotowanie raportu, a zamawiającemu jego późniejszą weryfikację.

Author: Materiały dowodowe są przedmiotem odrębnego załącznika

### 3.3. Charakter ocen

Raport wykorzystuje jednolity sposób przedstawiania stopnia spełnienia wymagań.

Ocena każdego wymagania wskazuje, czy rozwiązanie:

- spełnia wymaganie;

> - częściowo spełnia wymaganie;

Reviewer: Może warto doprecyzować, co oznacza ocena „częściowo spełnia wymaganie”. Proponuję, aby w takim przypadku raport wskazywał, która część wymagania nie została spełniona, jakie ma to konsekwencje dla użytkowników oraz - jeśli jest to możliwe - sposób i planowany termin usunięcia niezgodności. Pozwoli to zamawiającemu lepiej ocenić rzeczywisty wpływ niezgodności na dostępność rozwiązania.

- nie spełnia wymagania;
- wymaganie nie dotyczy rozwiązania.

Ocena stanowi syntetyczne podsumowanie opisu przedstawionego dla danego wymagania i nie zastępuje opisu sposobu realizacji wymagania.

### 3.4. Materiały dowodowe

Raport wskazuje materiały dowodowe stanowiące podstawę przedstawionych ocen.

Materiały dowodowe mogą stanowić załączniki do raportu lub być udostępniane zamawiającemu w inny uzgodniony sposób.

### 3.5. Wykorzystanie raportu

Raport stanowi podstawowy dokument komunikacji technicznej pomiędzy wykonawcą i zamawiającym dotyczący dostępności rozwiązania.

> Raport jest wykorzystywany przez zamawiającego podczas oceny rozwiązania i podejmowania decyzji w procesie jego odbioru, ale nie zastępuje procedury odbioru ani decyzji o odbiorze rozwiązania.

Reviewer: Warto rozważyć doprecyzowanie sposobu weryfikacji raportu przez zamawiającego. Czy zamawiający powinien przeprowadzić własne testy, audyt lub w inny sposób zweryfikować przedstawione przez wykonawcę oceny i materiały dowodowe? Byłoby to szczególnie pomocne dla podmiotów, które nie posiadają własnych specjalistów w zakresie dostępności cyfrowe

Reviewer 2: "Czy zamawiający powinien przeprowadzić własne testy, audyt lub w inny sposób zweryfikować przedstawione przez wykonawcę oceny" - 
W przypadku  innych zamawianych produktów nie weryfikujemy, wierzymy oświadczeniom wykonawcy. Czy kupując np. opony testujemy je. Dodatkowe koszty.

> ---

Reviewer 3: Opis PR wskazuje, że projekt rozwija projekt zalecenia Deklaracja dostawcy o dostępności cyfrowej produktu lub usługi, ale żaden z dokumentów PR się do niego nie odnosi, a tamten dokument nie jest modyfikowany ani wycofywany. Oba projekty nakładają na organizację obowiązek żądania od wykonawcy dokumentu o zgodności przy odbiorze rozwiązania: tamten deklaracji dostępności, ten raportu stopnia zgodności. Po przyjęciu obu zaleceń organizacje miałyby dwa równoległe, nieuzgodnione obowiązki. 

Czy intencją jest zastąpienie tamtego projektu tym zaleceniem, czy ich współistnienie? Jeżeli zastąpienie, tamten dokument powinien zostać wycofany lub oznaczony jako zastąpiony. Jeżeli współistnienie, relacja między deklaracją a raportem powinna być opisana w obu dokumentach.

id: raportowanie-stopnia-zgodnosci-rozwiazania
title: Raportowanie zamawiającemu stopnia zgodności rozwiązania z wymaganiami dostępności cyfrowej
description: Zalecenie dotyczące przekazywania zamawiającemu informacji o stopniu zgodności rozwiązania cyfrowego z wymaganiami dostępności oraz materiałach stanowiących podstawę tej oceny.
sidebar_label: Zalecenie

> sidebar_position: 1 

Reviewer 3: ```suggestion
sidebar_position: 1
```

keywords: [raport stopnia zgodności,dostępność cyfrowa,zgodność,zamówienia publiczne,odbiór rozwiązania]
tags: [raport stopnia zgodności,dostępność cyfrowa,zgodność,zamówienia publiczne,odbiór rozwiązania]
opracowanie: Stefan Wajda
data_zgloszenia: 31 lipca 2026 r.
ostatnia_aktualizacja: 31 lipca 2026 r.
wersja_robocza: true
---

> ## 1. Cel

Reviewer 3: w https://siec-dostepnosci-cyfrowej.github.io/sdc/generator-zalecen nie ma celu, czy to zamierzona zmiana?

Zapewnienie skutecznej komunikacji pomiędzy wykonawcą a zamawiającym dotyczącej dostępności rozwiązania cyfrowego poprzez przekazywanie raportu przedstawiającego właściwości rozwiązania mające znaczenie dla jego dostępności, ocenę stopnia spełnienia wymagań oraz materiały stanowiące podstawę przedstawionych ocen.

---

## 2. Zalecenie

Organizacja wymaga od wykonawców przekazania raportu stopnia zgodności rozwiązania z wymaganiami dostępności cyfrowej.

Raport przedstawia właściwości dostępności rozwiązania, opisuje sposób realizacji mających zastosowanie wymagań dostępności, określa stopień ich spełnienia oraz wskazuje materiały dowodowe stanowiące podstawę przedstawionych ocen.

---

## 3. Rekomendacje

### 3.1. Określanie obowiązku przekazania raportu

Organizacja określa w dokumentach zamówienia obowiązek przekazania raportu stopnia zgodności jako jednego z rezultatów realizacji zamówienia, jeżeli przedmiot zamówienia obejmuje rozwiązanie cyfrowe podlegające wymaganiom dostępności.

### 3.2. Zakres raportu

Raport obejmuje wszystkie mające zastosowanie wymagania dostępności objęte zakresem zamówienia.

> 
> Dla każdego wymagania raport zawiera:
> 
> - ocenę stopnia spełnienia wymagania;
> - opis sposobu realizacji wymagania, właściwości rozwiązania mających znaczenie dla jego dostępności oraz znanych ograniczeń;
> - wskazanie materiałów dowodowych stanowiących podstawę oceny.

Reviewer 3: ```suggestion
Dla każdego wymagania raport zawiera:

- ocenę stopnia spełnienia wymagania;
- opis sposobu realizacji wymagania, właściwości rozwiązania mających znaczenie dla jego dostępności oraz znanych ograniczeń;
- planowane działania dotyczące stwierdzonych niezgodności, jeżeli wymaganie nie zostało spełnione w pełni;
- wskazanie materiałów dowodowych stanowiących podstawę oceny.
```

Raport stopnia zgodności umożliwia wykonawcy przekazanie tej wiedzy w jednolitej, uporządkowanej i udokumentowanej postaci. Nie zastępuje on raportów z testów ani dokumentacji technicznej, lecz syntetycznie przedstawia stan zgodności rozwiązania oraz wskazuje materiały dowodowe stanowiące podstawę przedstawionych ocen.

Raport stopnia zgodności nie powinien być postrzegany jako dodatkowy obowiązek dokumentacyjny wykonawcy. Stanowi uporządkowany opis właściwości dostępności rozwiązania, ułatwiający komunikację pomiędzy wykonawcą i zamawiającym podczas odbioru rozwiązania oraz jego dalszego rozwoju i utrzymania.

Dobrze przygotowany raport pozwala wielokrotnie wykorzystywać zgromadzoną wiedzę podczas kolejnych wdrożeń, odbiorów, rozwoju produktu oraz współpracy z innymi zamawiającymi.

### 4.2. Europejski model raportowania zgodności

> Europejski model oceny zgodności stron internetowych i aplikacji mobilnych podmiotów publicznych odszedł od przedstawiania zgodności wyłącznie jako stanu „zgodne” lub „niezgodne”. Dyrektywa (UE) 2016/2102 oraz decyzja wykonawcza Komisji (UE) 2018/1523 przyjęły model raportowania stopnia spełnienia poszczególnych wymagań dostępności. Pozwala on przedstawić, które wymagania są spełnione, które są spełnione częściowo, które nie są spełnione oraz które nie mają zastosowania do ocenianego rozwiązania.

Reviewer 3: ```suggestion
Europejski model oceny zgodności stron internetowych i aplikacji mobilnych podmiotów publicznych odszedł od przedstawiania zgodności wyłącznie jako stanu „zgodne” lub „niezgodne”. Dyrektywa (UE) 2016/2102 oraz decyzja wykonawcza Komisji (UE) 2018/1523 wprowadziły w oświadczeniu w sprawie dostępności status zgodności obejmujący także zgodność częściową oraz wykaz treści niedostępnych. Ocena stopnia spełnienia poszczególnych wymagań, wskazująca wymagania spełnione, spełnione częściowo, niespełnione oraz niemające zastosowania, jest natomiast utrwalona w praktyce raportów zgodności takich jak VPAT/ACR.
```

Author: Propozycja zmienia sens tego fragmentu. Dla uzasadnienia nie ma znaczenia, że w praktyce raportów VPAT/ACR mamy nie "zerojedynkową" ocenę zgodności (jak w WCAG). 
Treść tego fragmentu mówi o tym, że model przyjęty w prawie europejskim nie jest zerojedynkowy i że to właśnie powala/uzasadnia na opisywanie stanu zgodności zestawem wartości: zgodnie, częściowo zgodne, niezgodne (spełnia, częściowo  spełnia, nie spełnia), nie dotyczy (nie ma zastosowania).

Reviewer 3: Zgoda co do intencji: chodzi o to, że model europejski nie jest zerojedynkowy i to uzasadnia skalę wielowartościową. Zastrzeżenie dotyczy tylko poziomu, na którym ten model działa. Decyzja 2018/1523 określa status zgodności dla całego rozwiązania (zgodna, częściowo zgodna, niezgodna) wraz z wykazem treści niedostępnych, w tym treści wyłączonych z zakresu stosowania przepisów; nie ustanawia raportowania stopnia spełnienia poszczególnych wymagań. Ocena per wymaganie jest rozwiązaniem przyjętym w tym zaleceniu i warto to tak przedstawić, zamiast przypisywać ją dyrektywie i decyzji.

> Zalecenie wykorzystuje ten model jako podstawę raportowania stopnia zgodności rozwiązania z wymaganiami dostępności cyfrowej.

Reviewer 3: ```suggestion
Zalecenie łączy oba podejścia: przyjmuje ocenę stopnia zgodności zamiast oceny binarnej oraz odnosi ją do poszczególnych wymagań dostępności cyfrowej.
```

Author: Tu nie ma połączenia obu podejść. W WCAG wymaganie jest spełnione lub nie spełnione. Jeśli nie ma zastosowania, uznaje się je za spełnione.

Reviewer 3: To zdanie odwołuje się do modelu opisanego w poprzednim akapicie, więc rozstrzygnięcie zapadnie w tamtym wątku

Stosowanie raportu stopnia zgodności zapewnia zamawiającemu jednolitą i porównywalną informację o stanie zgodności rozwiązań oferowanych przez różnych wykonawców oraz ułatwia podejmowanie decyzji opartych na udokumentowanych informacjach.

Raport zmienia charakter współpracy pomiędzy wykonawcą i zamawiającym. Zamiast ograniczać się do potwierdzenia zgodności lub przedstawienia listy stwierdzonych niezgodności, umożliwia wykonawcy przedstawienie właściwości dostępności rozwiązania, jego mocnych stron, znanych ograniczeń oraz planowanych kierunków rozwoju. Zamawiający otrzymuje informacje pozwalające zrozumieć rozwiązanie oraz zweryfikować każde istotne stwierdzenie na podstawie wskazanych materiałów dowodowych.

Raport umożliwia prowadzenie merytorycznej rozmowy o właściwościach dostępności rozwiązania zamiast ograniczania komunikacji do potwierdzania zgodności lub wskazywania niezgodności.

### 4.5. Znaczenie raportu

> Raport stopnia zgodności jest technicznym opisem właściwości rozwiązania mających znaczenie dla jego dostępności, przygotowanym przez producenta, dostawcę lub usługodawcę na potrzeby zamawiającego. Jego celem jest przekazanie uporządkowanej wiedzy o sposobie realizacji mających zastosowanie wymagań dostępności, stopniu ich spełnienia, ograniczeniach rozwiązania oraz materiałach umożliwiających weryfikację przedstawionych informacji.

Reviewer 3: W dokumencie i w specyfikacji strona sporządzająca raport jest nazywana niejednolicie. Sekcje 2 i 3 mówią o wykonawcy, sekcja 4.5 o producencie, dostawcy lub usługodawcy; specyfikacja w sekcjach 1.1, 1.2 i 6 mówi o producencie, dostawcy lub usługodawcy, a w 2.3 o wykonawcy. Czy to synonimy, czy różne role, na przykład wykonawca przekazuje raport sporządzony przez producenta? Jeżeli synonimy, proszę o ujednolicenie nazewnictwa w obu dokumentach; jeżeli różne role, ich relacja powinna być zdefiniowana.

Author: Zalecenie nie rozstrzyga, kto sporządza raport. Mówi o tym, że "organizacja wymaga od wykonawców przekazania raportu stopnia zgodności rozwiązania z wymaganiami dostępności cyfrowej". W tym kontekście "wykonawca" = strona umowy (kontrahent). Być może tu trzeba użyć innego słowa.

Reviewer 3: Rozumiem, to rozróżnienie ma sens. Skoro tak, zamiast ujednolicenia proponuję nazwać tę relację wprost przy pierwszym wystąpieniu raportu, wtedy przemienne używanie obu określeń przestaje wyglądać na niekonsekwencję. :)

Reviewer 3: dodalam propozycję w linii 23 - do rozważenia

Dzięki temu raport staje się przewodnikiem po dostępności rozwiązania oraz podstawowym dokumentem komunikacji technicznej pomiędzy wykonawcą i zamawiającym.

Każda ocena przedstawiona w raporcie powinna wynikać z opisu sposobu realizacji wymagania i wskazywać stopień jego spełnienia. Opis powinien wyjaśniać zastosowane rozwiązania, znane ograniczenia oraz wskazywać materiały dowodowe stanowiące podstawę oceny. Jeżeli producent, dostawca lub usługodawca nie potrafi opisać sposobu realizacji wymagania ani wskazać materiałów stanowiących podstawę przedstawionej oceny, oznacza to, że nie dysponuje wiedzą pozwalającą wiarygodnie określić stopień zgodności rozwiązania.

Raport nie zastępuje deklaracji zgodności, raportów z testów ani dokumentacji technicznej. Stanowi ich syntetyczne uzupełnienie, umożliwiające zamawiającemu zrozumienie właściwości rozwiązania mające znaczenie dla jego dostępności oraz podjęcie świadomej decyzji w procesie jego odbioru.

---

> ## 5. Podstawy prawne

Reviewer 3: ```suggestion
## 5. Podstawy prawne

- Ustawa z dnia 4 kwietnia 2019 r. o dostępności cyfrowej stron internetowych i aplikacji mobilnych podmiotów publicznych.
- Ustawa z dnia 11 września 2019 r. Prawo zamówień publicznych, w szczególności art. 100 ust. 1 oraz art. 106.
```

---

> ## 6. Powiązane dokumenty

Reviewer 3: ```suggestion
## 5. Źródła i opracowania

## 6. Historia wersji

## Powiązane dokumenty
```

> ## 6. Powiązane dokumenty

Reviewer 3: numeracja do zmiany, dodałam brakujace sekcje

---
id: raportowanie-stopnia-zgodnosci-rozwiazania
title: Raportowanie zamawiającemu stopnia zgodności rozwiązania z wymaganiami dostępności cyfrowej
description: Zalecenie dotyczące przekazywania zamawiającemu informacji o stopniu zgodności rozwiązania cyfrowego z wymaganiami dostępności oraz materiałach stanowiących podstawę tej oceny.
sidebar_label: Zalecenie
sidebar_position: 1
keywords: [raport stopnia zgodności,dostępność cyfrowa,zgodność,zamówienia publiczne,odbiór rozwiązania]
tags: [raport stopnia zgodności,dostępność cyfrowa,zgodność,zamówienia publiczne,odbiór rozwiązania]
opracowanie: Stefan Wajda
data_zgloszenia: 31 lipca 2026 r.
ostatnia_aktualizacja: 31 lipca 2026 r.
wersja_robocza: true
---

## 1. Cel

Zapewnienie skutecznej komunikacji pomiędzy wykonawcą a zamawiającym dotyczącej dostępności rozwiązania cyfrowego poprzez przekazywanie raportu przedstawiającego właściwości rozwiązania mające znaczenie dla jego dostępności, ocenę stopnia spełnienia wymagań oraz materiały stanowiące podstawę przedstawionych ocen.

---

## 2. Zalecenie

> Organizacja wymaga od wykonawców przekazania raportu stopnia zgodności rozwiązania z wymaganiami dostępności cyfrowej.

Reviewer 3: ```suggestion
Organizacja wymaga od wykonawców przekazania raportu stopnia zgodności rozwiązania z wymaganiami dostępności cyfrowej. Raport przekazuje wykonawca jako strona umowy; raport może być sporządzony przez producenta, dostawcę lub usługodawcę rozwiązania.
```

Raport stopnia zgodności umożliwia wykonawcy przekazanie tej wiedzy w jednolitej, uporządkowanej i udokumentowanej postaci. Nie zastępuje on raportów z testów ani dokumentacji technicznej, lecz syntetycznie przedstawia stan zgodności rozwiązania oraz wskazuje materiały dowodowe stanowiące podstawę przedstawionych ocen.

Raport stopnia zgodności nie powinien być postrzegany jako dodatkowy obowiązek dokumentacyjny wykonawcy. Stanowi uporządkowany opis właściwości dostępności rozwiązania, ułatwiający komunikację pomiędzy wykonawcą i zamawiającym podczas odbioru rozwiązania oraz jego dalszego rozwoju i utrzymania.

Dobrze przygotowany raport pozwala wielokrotnie wykorzystywać zgromadzoną wiedzę podczas kolejnych wdrożeń, odbiorów, rozwoju produktu oraz współpracy z innymi zamawiającymi.

### 4.2. Europejski model raportowania zgodności

> Europejski model oceny zgodności stron internetowych i aplikacji mobilnych podmiotów publicznych odszedł od przedstawiania zgodności wyłącznie jako stanu „zgodne” lub „niezgodne”. Dyrektywa (UE) 2016/2102 oraz decyzja wykonawcza Komisji (UE) 2018/1523 przyjęły model raportowania stopnia spełnienia poszczególnych wymagań dostępności. Pozwala on przedstawić, które wymagania są spełnione, które są spełnione częściowo, które nie są spełnione oraz które nie mają zastosowania do ocenianego rozwiązania.

Reviewer 3: ```suggestion
Europejski model oceny zgodności stron internetowych i aplikacji mobilnych podmiotów publicznych odszedł od przedstawiania zgodności wyłącznie jako stanu „zgodne” lub „niezgodne”. Dyrektywa (UE) 2016/2102 oraz decyzja wykonawcza Komisji (UE) 2018/1523 przyjęły model, w którym zgodność może być także częściowa, a treści mogą być wyłączone z zakresu mających zastosowanie wymagań. Zalecenie przenosi to podejście na poziom poszczególnych wymagań: raport przedstawia, które wymagania są spełnione, które są spełnione częściowo, które nie są spełnione oraz które nie mają zastosowania do ocenianego rozwiązania.
```
